frontend/src/components/Common/ReportDialog/index.tsx

```diff
@@ -107,6 +119,29 @@ const ReportDialog = memo(
       theme,
     ]);
 
+    const renderedPdfLoadingProgressBar = useMemo(() => {
+      if (!documentIsLoading) {
+        return null;
+      }
+      return (
+        <Box className={classes.documentLoadingContainer}>
+          <CircularProgress size={100} />
+          <Typography
+            className={classes.documentLoadingText}
+            variant="body1"
+            component="span"
+          >
+            {t('Loading document...')}
```

Comment:
Let's remove the `...` from the trabslation function

Reply:
Updated